app/webpacker/styles/app_task_list.scss

```diff
@@ -1,23 +1,30 @@
 .app-task-list {
+  padding-left: 0;
   list-style: none;
 }
 
 .app-task-list__items {
-  padding-left: 30px;
+  padding-left: 0;
   margin-bottom: 60px;
   list-style: none;
 }
 
 .app-task-list__item {
-  border-bottom: 1px solid #b1b4b6;
+  $border: 1px solid #b1b4b6;
```

Comment:
Does this need to be in a variable too?

Reply:
what do you mean?

Reply:
Does #b1b4b6 need to be declared as a variable? I've seen it cause some warnings in the console and just wondered if that would be a good thing to do.